bundles/net.tourbook/icon-factory/common.svg

```diff
@@ -7973,5 +7973,41 @@
        id="rect17705-1-1-8-1"
        d="m 184.2774,69.713507 3.72262,-3.53224 3.72255,3.53224"
        style="display:inline;fill:#d0ec86;fill-opacity:1;stroke:#435800;stroke-width:0.8;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1;paint-order:normal" />
+    <rect
```

Comment:
> When you draw between 2 pixels then the exported image is blurred, mostly. **Always** draw straight lines at the pixel border
> 

Thanks for the tip, I didn't know. I will fix and also check my other previous images to see if I did the same mistake

Reply:
Fixed
![image](https://user-images.githubusercontent.com/25235665/117095507-c675c680-ad23-11eb-8183-edfc453c9086.png)

Reply:
> 
> 
> > When you draw between 2 pixels then the exported image is blurred, mostly. **Always** draw straight lines at the pixel border
> 
> Thanks for the tip, I didn't know. I will fix and also check my other previous images to see if I did the same mistake

Because I'm currently also modifying the common.svg file to draw dark images and 2 modified svg files cannot be easily merged, we should separate your and my svg files otherwise there could be always conflicts which are time intensive to solve. 

Could you draw your images into your "private" svg file ?

Reply:
> Could you draw your images into your "private" svg file ?

Today I merged with my private svg file.

Reply:
It's better now when I zoom
![image](https://user-images.githubusercontent.com/25235665/117360282-1d0c0d80-aea8-11eb-894f-27364956ea53.png)

Reply:
Also, I put the labels outside the button and inverted the icons:
- To enable the radio buttons, the "opened lock" image is displayed to show to the user that the action is to "unlock"
And vice versa.

Do you think it's more intuitive ? or that it was better before ?

Reply:
> 
> 
> Also, I put the labels outside the button and inverted the icons:
> 
>     * To enable the radio buttons, the "opened lock" image is displayed to show to the user that the action is to "unlock"
>       And vice versa.
> 
> 
> Do you think it's more intuitive ? 

No

>or that it was better before ?

No

The button with icon is more intuitive vor me and easier accessable because of a wider button

Reply:
> 
> 
> It's better now when I zoom
> 
> ![image](https://user-images.githubusercontent.com/25235665/117373897-a711a180-aebb-11eb-86a4-8db3cf69a54d.png)

In your initial image, there where still some blurring, yellow markings, but it seems you have fixed it

![mt lock image](https://user-images.githubusercontent.com/1283445/117403298-3a70c400-af08-11eb-905b-dda0a02cc827.png)

In Inkscape I often fix positons in the x/y decimals, you can also do it with the mousewheel or cursor up/down keys

![mt decimals](https://user-images.githubusercontent.com/1283445/117403350-51afb180-af08-11eb-9da4-a2260a629b12.png)

Reply:
> > Do you think it's more intuitive ?
> 
> No
> 
> > or that it was better before ?
> 
> No

So what do you recommend since both of your answers are no ?

> 
> The button with icon is more intuitive vor me and easier accessable because of a wider button

I changed it out of the button because i realized that when I click on the button, because the label changes (smaller or longer text), the lock image placement changes too and I thought it would not be "good UI"

![image](https://user-images.githubusercontent.com/25235665/117489399-1c817e80-af5d-11eb-9c3c-4615e217d3b7.png)

Reply:
> 
> 
> > > Do you think it's more intuitive ?
> > 
> > 
> > No
> > > or that it was better before ?
> > 
> > 
> > No
> 
> So what do you recommend since both of your answers are no ?

Sorry, I've not read the 2nd question thoroughly and was wondering why you asked it differently. 
So a yes for "or that it was better before ?"

Reply:
> changed it out of the button because i realized that when I click on the button, because the label changes (smaller or longer text), the lock image placement changes too and I thought it would not be "good UI"
> 
> ![image](https://user-images.githubusercontent.com/25235665/117489399-1c817e80-af5d-11eb-9c3c-4615e217d3b7.png)
> 
> Placing the label outside the button doesn't create this issue
> <img alt="" width="40%" height="40%" src="https://user-images.githubusercontent.com/25235665/117489573-4e92e080-af5d-11eb-8691-cbd8adacf197.png">
> 
> let me know what you prefer


A SWT button cannot easily left align image and text, just a text or just an image would be OK

![swt button with images left aligned](https://user-images.githubusercontent.com/1283445/117527627-b54fe280-afcd-11eb-84ce-af2dc4fdc713.png)

With some more work it could be achieved

![easy import UI](https://user-images.githubusercontent.com/1283445/117527670-0fe93e80-afce-11eb-8cca-2d27b63cbf4f.png)

These is my test code

![easy import code](https://user-images.githubusercontent.com/1283445/117527681-1bd50080-afce-11eb-9b6b-ed6444bb0759.png)

The default easy import dialog is showing image and text in the center, this would be OK also for the lock button

![easy import UI 2](https://user-images.githubusercontent.com/1283445/117527748-900fa400-afce-11eb-833d-93d175b9ae76.png)

Reply:
> Sorry, I've not read the 2nd question thoroughly and was wondering why you asked it differently.
> So a yes for "or that it was better before ?"

Ok, I have reverted to the previous behavior:
![image](https://user-images.githubusercontent.com/25235665/117724713-ae92bc80-b1d3-11eb-949e-ed57253167da.png)

Reply:
> These is my test code
> 
> ![easy import code](https://user-images.githubusercontent.com/1283445/117527681-1bd50080-afce-11eb-9b6b-ed6444bb0759.png)
> 
Good to know even though it's not a very "pretty" solution

Reply:
>Good to know even though it's not a very "pretty" solution

This is a bug in SWT, or it's not yet fully implemented, that image and text are not left aligned.

You may try to add programmatically some spaces at the end, that the "Lock" and "Unlock" texts have the same number of characters.